### PR TITLE
Issue #10 fix with test

### PR DESCRIPTION
**Pull request by fporcaro**
I added the test and the fix.  If you keep the test without the fix, you will see the problem and the fix for Mock == seems to work and didn't break the other tests.

### CODE REVIEW

**Comment by ghost** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 12 tests, with 0 failures (0 unexpected) in 1.502 (1.513) seconds</td>
    </tr>
  </tbody>
</table>

## Current coverage for Mocker is `94.81%`
Files changed | - | - 
--- | --- | ---
Mock.swift | `95.59%` | :white_check_mark:

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by fporcaro** [comment]
I saw the Travis build failed, but looking at the fastlane output, and it looks like there are some lint problems, but none of them are in the code for the unit test I added, so I'm not sure how it ever built with Travis before?

File: MockerTests/MockerTests.swift [hunk 1]
```diff
@@ -228,4 +228,18 @@ final class MockerTests: XCTestCase {
         Mocker.ignore(ignoredURL)
         XCTAssert(MockingURLProtocol.canInit(with: URLRequest(url: ignoredURL)) == false)
     }
+    
+    func testComposedURLMatch() {
+        let composedURL = URL(fileURLWithPath: "resource", relativeTo: URL(string: "https://host.com/api/"))
+        let simpleURL = URL(string: "https://host.com/api/resource")
```
Comment (AvdLee) [inline MockerTests/MockerTests.swift:259]: You can force unwrap this one which enables you to remove the XCTFail on line 241

File: MockerTests/MockerTests.swift [hunk 2]
```diff
@@ -228,4 +228,18 @@ final class MockerTests: XCTestCase {
         Mocker.ignore(ignoredURL)
         XCTAssert(MockingURLProtocol.canInit(with: URLRequest(url: ignoredURL)) == false)
     }
+    
+    func testComposedURLMatch() {
```
Comment (AvdLee) [inline MockerTests/MockerTests.swift:257]: Can you add a describing documentation starting with "It should .. "

**Review by AvdLee — CHANGES_REQUESTED** [review]
Great work! I see that CI is failing, but your tests are succeeding. I will look into the CI issues, you can ignore them.

For this PR; would you mind adding a test for URLs including a query matching?

**Comment by AvdLee** [comment]
https://github.com/WeTransfer/Mocker/pull/14 Will fix the unwraps, you can merge that in once it's merged.

**Comment by AvdLee** [comment]
You can merge #14 in now!

**Comment by fporcaro** [comment]
Changes should be made and merged into my branch.  Looks like the CI passed.  Let me know what else you need from me.

**Review by AvdLee — APPROVED** [review]

**Pull request merged**